src/k8s-configuration/azext_k8s_configuration/consts.py

```diff
@@ -7,29 +7,53 @@
 
 # API VERSIONS -----------------------------------------
 
-SOURCE_CONTROL_API_VERSION = '2021-03-01'
-FLUXCONFIG_API_VERSION = '2021-11-01-preview'
+SOURCE_CONTROL_API_VERSION = "2021-03-01"
+FLUXCONFIG_API_VERSION = "2022-01-01-preview"
 EXTENSION_API_VERSION = "2021-09-01"
 
 # ERROR/HELP TEXT DEFINITIONS -----------------------------------------
 
-KUSTOMIZATION_REQUIRED_VALUES_MISSING_ERROR = "Error! Kustomization definition is invalid, required values {} not found"
-KUSTOMIZATION_REQUIRED_VALUES_MISSING_HELP = "Add the required values to the Kustomization object and try again"
+KUSTOMIZATION_REQUIRED_VALUES_MISSING_ERROR = (
+    "Error! Kustomization definition is invalid, required values {} not found"
+)
+KUSTOMIZATION_REQUIRED_VALUES_MISSING_HELP = (
+    "Add the required values to the Kustomization object and try again"
+)
 
 REPOSITORY_REF_REQUIRED_VALUES_MISSING_ERROR = "Error! Repository reference is invalid"
-REPOSITORY_REF_REQUIRED_VALUES_MISSING_HELP = "Specifying one of [--branch, --tag, --semver, --commit] is required"
+REPOSITORY_REF_REQUIRED_VALUES_MISSING_HELP = (
+    "Specifying one of [--branch, --tag, --semver, --commit] is required"
+)
 
 REPOSITORY_REF_TOO_MANY_VALUES_ERROR = "Error! Repository reference is invalid"
-REPOSITORY_REF_TOO_MANY_VALUES_HELP = "Specifying more than one repository ref argument is invalid"
-
-GIT_REPOSITORY_REQUIRED_VALUES_MISSING_ERROR = "Error! Required property '{}' was not specified for kind 'GitRepository'"
-GIT_REPOSITORY_REQUIRED_VALUES_MISSING_HELP = "Add missing required property and try again"
+REPOSITORY_REF_TOO_MANY_VALUES_HELP = (
+    "Specifying more than one repository ref argument is invalid"
+)
+
+REQUIRED_VALUES_MISSING_ERROR = (
+    "Error! Required properties [{}] were not specified for kind '{}'"
+)
+REQUIRED_VALUES_MISSING_HELP = "Add the missing required properties and try again"
```

Comment:
nit: to be consistent, do you want to have all of the string messages be inside parentheses?

Reply:
It is a formatting thing. This is done by auto-formatter for strings that are too long